Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/WorldAnchorManager.cs

```diff
@@ -0,0 +1,608 @@
+        /// If a GameObject reference can be found, the anchor component will be removed.
+        /// </summary>
+        /// <param name="anchorName">The name of the anchor to remove from the anchor store.</param>
+        public void RemoveAnchor(string anchorName)
+        {
+            if (string.IsNullOrEmpty(anchorName))
+            {
+                Debug.LogErrorFormat("[WorldAnchorManager] Invalid anchor \"{0}\"! Try removing anchor by GameObject.", anchorName);
+                if (anchorDebugText != null)
+                {
+                    anchorDebugText.text += string.Format("\nInvalid anchor \"{0}\"! Try removing anchor by GameObject.", anchorName);
+                }
+                return;
+            }
+
+            RemoveAnchor(anchorName, null);
+        }
+
+        /// <summary>
+        /// Removes the anchor from the game object and deletes the anchor
+        /// from the anchor store.
+        /// </summary>
+        /// <param name="anchorName">Name of the anchor to remove from the anchor store.</param>
+        /// <param name="gameObjectToUnanchor">GameObject to remove the anchor from.</param>
+        private void RemoveAnchor(string anchorName, GameObject gameObjectToUnanchor)
+        {
+            if (string.IsNullOrEmpty(anchorName) && gameObjectToUnanchor == null)
+            {
+                Debug.LogWarning("Invalid Remove Anchor Request!");
+                return;
+            }
+
+#if !UNITY_WSA || UNITY_EDITOR
+            Debug.LogWarning("World Anchor Manager does not work for this build. RemoveAnchor will not be called.");
+#else
+            // This case is unexpected, but just in case.
+            if (AnchorStore == null)
+            {
+                Debug.LogWarning("[WorldAnchorManager] RemoveAnchor called before anchor store is ready.");
+            }
+
+            LocalAnchorOperations.Enqueue(
+                new AnchorAttachmentInfo
+                {
+                    AnchoredGameObject = gameObjectToUnanchor,
+                    AnchorName = anchorName,
+                    Operation = AnchorOperation.Delete
+                });
+#endif
+        }
+
+        /// <summary>
+        /// Removes all anchors from the scene and deletes them from the anchor store.
+        /// </summary>
+        public void RemoveAllAnchors()
+        {
+#if !UNITY_WSA || UNITY_EDITOR
+            Debug.LogWarning("World Anchor Manager does not work for this build. RemoveAnchor will not be called.");
+#else
+            // This case is unexpected, but just in case.
+            if (AnchorStore == null)
+            {
+                Debug.LogWarning("[WorldAnchorManager] RemoveAllAnchors called before anchor store is ready.");
+            }
+
+            var anchors = FindObjectsOfType<WorldAnchor>();
+
+            if (anchors == null)
+            {
+                return;
+            }
+
+            for (var i = 0; i < anchors.Length; i++)
```

Comment:
nit: why var when int is just as few characters? :)

Reply:
Old code, changed.